src/Azure.Functions.Cli/Helpers/DotnetHelpers.cs

```diff
@@ -86,6 +86,7 @@ public static async Task DeployDotnetFunction(string templateName, string functi
         private static string GetTemplateShortName(string templateName) => templateName.ToLowerInvariant() switch
         {
             "blobtrigger" => "blob",
+            "eventgridblobtrigger" => "egblob",
```

Comment:
I think we want this to match the short name here: https://github.com/Azure/azure-functions-templates/pull/1519/files#diff-f46c6528d07abea982faaa52631c315020e217f2e08caaada8997b3e53cbd237R12
```suggestion
            "eventgridblobtrigger" => "eventgridblob",
```

Reply:
Updated